magefile.go

```diff
@@ -34,6 +34,7 @@ var Aliases = map[string]interface{}{
 
 func GenProto() {
 	sh.Run("protoc", "-Iapi", "-Iapi/vendor", "--go_out=./api", "--go-grpc_out=./api", "--go_opt=paths=source_relative", "--go-grpc_opt=paths=source_relative", "api/shipper.proto")
+	common.Fmt()
```

Comment:
The least surprising thing to do is to only format the directory containing the generated files. I don't think it matters too much though, since you'd have to run format to get CI to pass anyway.